beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2798,6 +2831,9 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             notify_execution_layer,
         )?;
 
+        //TODO(sean) error handling?
```

Comment:
Probably something that makes more sense in the original PR. I say we keep it as is here and change it only if required for something deneb specific.

Reply:
Yea I think the thing here was I was wondering if I should pass the error through `handle_block_error`, but that method just prints out a log. I think `publish_fn` handles the logging fine, there's only a couple error cases and they're logged or returned via API errors